src/main/java/com/google/sps/model/GeocodingClient.java

```diff
@@ -33,26 +35,35 @@
    * @throws GeocodingException A custom exception is thrown to signal an error pertaining to the
    *     Geocoding API.
    */
-  GeocodingResult getGeocodingResult(String address) throws GeocodingException;
+  List<GeocodingResult> getGeocodingResult(String address) throws GeocodingException;
 
   /**
    * Parses for coordinates in a resulting call to the Geocoding API.
    *
    * @param result A GeocodingResult returned from the Geocoding API.
    * @return A LatLng representing coordinates.
    */
-  public static LatLng getCoordinates(GeocodingResult result) {
-    return result.geometry.location;
+  public static LatLng getCoordinates(List<GeocodingResult> results) {
+    for (GeocodingResult result : results) {
+      List<AddressType> types = Arrays.asList(result.types);
+      for (AddressType type : types) {
```

Comment:
Perhaps use `isStreetAddress` here to simplify the check

Reply:
Ah unfortunately I want it to return in the middle of the loop so `isStreetAddress` wouldn't simplify the check. I think it's a naming issue so I changed it to `hasStreetAddress` instead.

Reply:
`RESOLVED`